src/main/java/net/tiagofar78/prisonescape/game/PrisonEscapeGame.java

```diff
@@ -762,6 +777,26 @@ public void policeInspectedPrisioner(String policeName, String prisionerName) {
         }
     }
 
+    /**
+     * @return true if door toggled<br>
+     *         false if door not toggled
```

Comment:
Delete this description as well.

We have just been leaving descriptions when the function returns 0, -1, -2, etc